projects/1.20-fabric/assets/dynamic-surroundings/dsurround/lang/zh_cn.json

```
@@ -0,0 +1,196 @@
+{
+  "dsurround.modname": "动态环绕",
+  "dsurround.newversion.update": "有更新可用",
+  "dsurround.newversion.downloadpage": "点击进入下载界面",
+  "dsurround.newversion.curseforge": "CurseForge",
+  "dsurround.newversion.modrinth": "Modrinth",
+  "dsurround.newversion.releasenotes": "更新日志",
+  "dsurround.newversion.releasenotespage": "点击查看更新日志",
+  "dsurround.mod.link.sources": "源代码存储库",
+  "dsurround.mod.link.credits": "声音资源的版权和归属",
+  "dsurround.mod.link.clothapi": "嵌入: Cloth API (Fabric)",
+  "dsurround.mod.link.nashorn": "嵌入: OpenJDK Nashorn JavaScript Engine",
+  "dsurround.sound.caption.ice": "冰：碎裂",
+  "dsurround.sound.caption.wind": "风：吹过",
+  "dsurround.sound.caption.soulsand": "狂笑声",
+  "dsurround.sound.caption.bird": "鸟：叽叽喳喳",
+  "dsurround.sound.caption.woodpecker": "啄木鸟：啄木头",
+  "dsurround.sound.caption.raptor": "猛禽：唳叫",
+  "dsurround.sound.caption.crow": "乌鸦：嘶鸣",
+  "dsurround.sound.caption.owl": "猫头鹰：鸣叫",
+  "dsurround.sound.caption.loon": "潜鸟：尖叫",
+  "dsurround.sound.caption.nightingale": "夜莺：歌唱",
+  "dsurround.sound.caption.rockfall": "岩石：滚落",
+  "dsurround.sound.caption.monstergrowl": "来势汹汹的咆哮",
+  "dsurround.sound.caption.waterdrop": "水：滴落",
+  "dsurround.sound.caption.seagulls": "海鸥：鸣叫",
+  "dsurround.sound.caption.howling": "狂风呼啸",
+  "dsurround.sound.caption.tummy": "胃：咕咕叫",
+  "dsurround.sound.caption.breathing": "沉重的呼吸声",
+  "dsurround.sound.caption.bees": "蜜蜂：嗡嗡",
+  "dsurround.sound.caption.insects": "昆虫：爬行",
+  "dsurround.sound.caption.pageflip": "书页：翻过",
+  "dsurround.sound.caption.rooster": "公鸡：打鸣",
+  "dsurround.sound.caption.anvil": "铁砧：敲响",
+  "dsurround.sound.caption.dog": "狗：嗥叫",
+  "dsurround.sound.caption.thunder": "轰雷贯耳",
+  "dsurround.sound.caption.bow.pull": "弓：拉动",
+  "dsurround.sound.caption.footsteps.water_through": "水：飞溅",
+  "dsurround.sound.caption.footsteps.brush_through": "草丛：窸窸窣窣",
+  "dsurround.sound.caption.footsteps.leaves_through": "树叶：沙沙作响",
+  "dsurround.sound.caption.footsteps.floor_squeak": "地板：吱嘎作响",
+  "dsurround.format.TimeOfDay": "第 %1$d 天, %2$d:%3$s %4$s (%5$s)",
+  "dsurround.format.AM": "AM",
+  "dsurround.format.PM": "PM",
+  "dsurround.format.NoSky": "没有天空",
+  "dsurround.format.Sunrise": "日出",
+  "dsurround.format.Sunset": "日落",
+  "dsurround.format.Daytime": "日间",
+  "dsurround.format.Nighttime": "夜间",
+  "dsurround.command.dsdump.success": "转存 %1$s \u00a72成功\u00a7r. 数据可以在 %2$s 里找到",

```

```suggestion
  "dsurround.command.dsdump.success": "转存 %1$s \u00a72成功\u00a7r。数据可以在 %2$s 里找到",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

通查完毕，已修改